### PR TITLE
Adding golden.hackclub.com

### PR DESCRIPTION
**Pull request by kriaga259**
## Description
Added Golden to hackclub.com.yaml
Golden is a Days of Service event part of the [Athena](https://athena.hackclub.com/events) intiative

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -1263,6 +1263,10 @@ globalcovid:
   ttl: 600
   type: CNAME
   value: globalcovid.netlify.com.
+golden:
+  - ttl: 600
+    type: CNAME
+    value: https://kriaga259.github.io/golden/.
```
Comment (Muirrum) [inline hackclub.com.yaml:1269]: DNS records can't have protocols (`https`) or paths (`/golden/`)

```suggestion
    value: kriaga259.github.io.
```

**Review by Muirrum — CHANGES_REQUESTED** [review]
addtl -- if this is an HQ event, site should probably not be hosted on a personal github account.

**Comment by revaga** [comment]
> addtl -- if this is an HQ event, site should probably not be hosted on a personal github account. suggest migrating repository to hackclub github org

how can we migrate to the hackclub github org?

**Comment by alx-alexpark** [comment]
Hi Reva 🔪!

You can use the repo creation request in the #hq-engineering channel bookmark under "Create service ticket". You can then migrate your git repo there.

File: hackclub.com.yaml [hunk 2]
```diff
@@ -1288,6 +1288,10 @@ globalcovid:
   ttl: 600
   type: CNAME
   value: globalcovid.netlify.com.
+golden:
+  - ttl: 600
+    type: CNAME
+    value: https://hackclub.github.io/golden/
```
Comment (alx-alexpark) [inline hackclub.com.yaml:1294]:
```suggestion
    value: hackclub.github.io.
```

**Review by alx-alexpark — CHANGES_REQUESTED** [review]
Hi! CNAME records cannot have a protocol or path in the record. You should assign a custom domain using the instructions [here](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site)!

**Comment by kriaga259** [comment]
Hi! Is it better now?

File: hackclub.com.yaml [hunk 3]
```diff
@@ -1288,6 +1288,10 @@ globalcovid:
   ttl: 600
   type: CNAME
   value: globalcovid.netlify.com.
+golden:
+  - ttl: 600
+    type: CNAME
+    value: golden.hackclub.dev.
```
Comment (alx-alexpark) [inline hackclub.com.yaml:1294]:
```suggestion
    value: cname.vercel-dns.com.
```

**Review by alx-alexpark — CHANGES_REQUESTED** [review]
Just one last thing: when deploying site from vercel you should cname to `cname.vercel-dns.com.` instead of the hackclub.dev domain. After this you should be all good!

**Review by alx-alexpark — APPROVED** [review]
yummy

**Pull request merged**